services/order.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+from django.utils import timezone
+
+from db.models import Order, Ticket, MovieSession, User
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None
+                 ) -> Order:
+    user = User.objects.get(username=username)
```

Comment:
Using `get` can raise a `DoesNotExist` exception if the user with the given username does not exist. Consider handling this exception to provide a more user-friendly error message.